### DISCUSSION TITLE
Display different sabers in HMD and on desktop

### DISCUSSION THREAD

**Discussion by nolanimations** [ToniMacaroni/SaberFactory#133] [Ideas]
(copied from my request in the commissions server, since I was advised to bring this idea here instead of making it a seperate mod):

Something that allows me to use different custom sabers in HMD and display other ones on desktop. You see, I have some AWESOME and VERY cool sabers in my custom saber folder, and they look extremely visually appealing on stream for example, much more than my regular custom sabers. BUT, they are absolute ATROCIOUS to focus with in my Virtual Reality headset. And of course, I care for the viewing experience of the loyal followers of my twitch.tv channel, so I need the best of both worlds :)

**Comment by ToniMacaroni** [comment]
This is something I wanted to do for a long time, but finally implemented in v3. It will be quite sophisticated as it will work with whole saber configurations (e.g. have the same saber in HMD and desktop but different trail configurations).
Im currently still on a break from modding, but some development updates regarding v3 will be coming soon.
